pages/index.vue

```diff
@@ -1,10 +1,10 @@
 <template>
-  <main>
+  <div>
```

Comment:
Same, `article`?

Reply:
Updated in https://github.com/Qiskit/qiskit.org/pull/3441/commits/8813034834a0b3ef57c06bdf5a74df881100161f